### DISCUSSION TITLE
Char Copy to PTR including Equipment Sets

### DISCUSSION THREAD

**Discussion by Erson33** [chromiecraft/chromiecraft#7408] [Suggestions]
Hello!

The char copy feature is fantastic to help to test on the PTR, however it does not copy equipment sets which are stored server side from what I have been able to tell. If it's possible, having these copy over with the char copy function would be very helpful!

Thank you,

Erson

**Comment by heyitsbench** [comment]
https://github.com/55Honey/Acore_CarbonCopy/issues/10
